lib/state/settings/reducer.js

```diff
@@ -41,6 +41,14 @@ const noteDisplay = (state = 'comfy', action) => {
   return action.noteDisplay;
 };
 
+const lineLength = (state = 'narrow', action) => {
```

Comment:
Don't think it really matters, but you could have made this a boolean settings since it's sort of a toggle vs. storing a string. But this will future-proof it in case we add another option in the future!

Reply:
Yes, I was wondering about that. I kind of assumed that was where we were going when it changed from a single “Full Width” toggle in your first iteration to a multiple choice. I do anticipate some kind of View Options dialog in the future 🙂

<img width="303" alt="screen shot 2018-09-04 at 21 30 43" src="https://user-images.githubusercontent.com/555336/45034463-99644800-b092-11e8-9338-729ce3e424e4.png">